packages/arb-node-core/cmd/arb-relay/arb-relay.go

```diff
@@ -141,19 +141,21 @@ func (ar *ArbRelay) Start(ctx context.Context) (chan bool, error) {
 	// connect returns
 	messages := make(chan broadcaster.BroadcastFeedMessage)
 	for _, client := range ar.broadcastClients {
-		for {
-			err = client.ConnectWithChannel(ctx, messages)
-			if err == nil {
-				break
-			}
-			logger.Warn().Err(err).
-				Msg("failed connect to sequencer broadcast, waiting and retrying")
-			select {
-			case <-ctx.Done():
-				return nil, errors.New("ctx cancelled broadcast client connect")
-			case <-time.After(5 * time.Second):
+		go (func() {
+			for {
+				err := client.ConnectWithChannel(ctx, messages)
+				if err == nil {
+					break
+				}
+				logger.Warn().Err(err).
```

Comment:
Would be good to include the url in this warning so we know which connection is failing